examples/with-react-native-web/packages/ui/src/index.tsx

```diff
@@ -1 +1 @@
-export { Button, type ButtonProps } from "./button";
+export { Button, type ButtonProps } from "./Button";
```

Comment:
This looks like it's causing the build to fail.

Reply:
Classic "works on my machine."